apps/api/src/app/tenant/usecases/get-tenant/get-tenant.command.ts

```diff
@@ -0,0 +1,9 @@
+import { EnvironmentCommand } from '@novu/application-generic';
+import { IsMongoId, IsNotEmpty, IsString } from 'class-validator';
+
+export class GetTenantCommand extends EnvironmentCommand {
+  @IsString()
+  @IsNotEmpty()
+  @IsMongoId()
+  id: string;
```

Comment:
What do you think about calling it tenantId here? So it remains clear when we use command.tenantId vs command.id in future uscases